lua/nvim-tree.lua

```diff
@@ -363,17 +363,17 @@ local function setup_autocommands(opts)
 
   create_nvim_tree_autocmd("BufReadPost", {
     callback = function()
-      if filters.config.filter_no_buffer or renderer.config.highlight_opened_files then
```

Comment:
Ohhh? I thought the if statement can still remain though, it'll still bring (possibly negligible) performance improvement ;)